overlord/devicestate/devicemgr.go

```diff
@@ -864,6 +873,11 @@ func (m *DeviceManager) RequestSystemAction(systemLabel string, action SystemAct
 		return fmt.Errorf("cannot load seed system: %v", err)
 	}
 
+	currentSys, _ := currentSeedSystem(m.state)
```

Comment:
should we pass currentSys to systemFromSeed and do the adding in there?  either way there's a TODO:UC20 there about building the action list that should go away now I suppose ?

Reply:
Updated